docs/network-flow-visibility.md

```diff
@@ -202,31 +203,54 @@ helm install theia build/charts/theia -n flow-visibility --create-namespace \
   --set=clickhouse.storageSize="2Gi"
 ```
 
-The ClickHouse TCP service is used by the Flow Aggregator. If you have changed the TCP port,
-please update the `databaseURL` following
-[Flow Aggregator Configuration](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#configuration-1).
+The ClickHouse TCP service is used by the Flow Aggregator. If you have changed
+the TCP port, please update the `clickHouse.databaseURL` in the Flow Aggregator
+Helm Chart values.
 
-The ClickHouse credentials are used in the Flow Aggregator. Them are also specified
-in `flow-aggregator.yml` as a Secret named `clickhouse-secret` as shown below.
-Please make the corresponding changes if you change the ClickHouse credentials.
+The ClickHouse credentials are used in the Flow Aggregator. If you have changed
+the ClickHouse credentials, please update the `clickHouse.connectionSecret` in the
+Flow Aggregator Helm Chart values.
 
-```yaml
-apiVersion: v1
-kind: Secret
-metadata:
-  labels:
-    app: flow-aggregator
-  name: clickhouse-secret
-  namespace: flow-aggregator
-stringData:
-  password: clickhouse_operator_password
-  username: clickhouse_operator
-type: Opaque
-```
-
-Please refer to the [Flow Aggregator Configuration](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#configuration-1)
+Please refer to the Flow Aggregator Helm Chart [README](../build/charts/flow-aggregator/README.md)
+and [Flow Aggregator Configuration](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#configuration-1)
 to learn about more ClickHouse configuration options in Flow Aggregator.
 
+##### ClickHouse Cluster
+
+From Theia v0.2.0, we support deploying ClickHouse cluster with data replication.
+A ClickHouse cluster consists of shards. Shards refer to the servers that contain
+different parts of the data, which provides a horizontal cluster scaling strategy.
+Each shard consists of one or more replica hosts to ensure reliability.
+
+To deploy ClickHouse cluster, please set `clickhouse.cluster.enable` to true and
+set `clickhouse.cluster.shards` and `clickhouse.cluster.replicas` per your
+requirement.
+
+ClickHouse uses [Apache ZooKeeper](https://zookeeper.apache.org/) for storing
+replicas meta information. By default, a ZooKeeper cluster using `emptyDir` with
+3 replicas will be deployed for ClickHouse cluster. Please make sure there are
+at least 3 Nodes available in your Kubernetes cluster. To use a customized
+ZooKeeper cluster, please refer to the
+[ZooKeeper setup instructions for ClickHouse](https://github.com/Altinity/clickhouse-operator/blob/master/docs/zookeeper_setup.md)
+and set `clickhouse.cluster.zookeeperHosts` to your ZooKeeper hosts.
+
+The default affinity allows only one ClickHouse instance per Node. To change the
```

Comment:
Do we need to remind user to have 4 nodes setup?  Since the default shard=2, replicas=2 and we have podDistribution setting MaxNumberPerNode=1.
Or use ShardAntiAffinity, since we already have at least 3 Nodes?

Reply:
I think basically `MaxNumberPerNode=1` would be recommended to achieve reliability. But I'm worry about the redundant statement on different number of Nodes would be confusing. I tried to rephrase the words here. Please take a look whether it is clearer. Appreciate any suggestion!

Reply:
I'm also thinking the number of nodes in zookeeper and clickchouse affinity will be confusing.
But I think current statement is clear enough for user to know where the numbers come from.